stdlib/2/sets.pyi

```diff
@@ -0,0 +1,60 @@
+# Stubs for sets (Python 2)
+from typing import Any, Callable, Hashable, Iterable, Iterator, Mapping, Optional, TypeVar, Union
+
+_T = TypeVar('_T')
+Setlike = Union['BaseSet', Iterable]
```

Comment:
Yes, we still need to add that to http://github.com/ambv/flake8-pyi. I'll get to it at some point.